multimodal/src/autogluon/multimodal/models/ft_transformer.py

```diff
@@ -594,12 +601,23 @@ def __init__(
             initialization="uniform",
         )
 
-        # init weights
+        # init tokenizer and head weights
         if self.numerical_feature_tokenizer:
             self.numerical_adapter.apply(init_weights)
         if self.categorical_feature_tokenizer:
             self.categorical_adapter.apply(init_weights)
         self.head.apply(init_weights)
+        # init transformer backbone from provided checkpoint
+        if pretrained and checkpoint_name:
+            if "https://" in checkpoint_name or is_s3_url(checkpoint_name):
```

Comment:
This can be simplified if using https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/utils/download.py#L31